CHANGELOG.md

```diff
@@ -6,14 +6,22 @@ This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 
 ## [Unreleased]
 
+## [0.6.1] - 2016-04-27
+### Added
+- Added cross_compiled utility to specify if a platform is cross_compiled
```

Comment:
Added is_cross_compiled? utility methods to specify if a platform is cross-compiled